### ISSUE TITLE
Not possible to monitor others' experiments (on /esarchive shared BSC-ES group filesystem)

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#265]
In GitLab by @dmanubens on Jul 13, 2017, 17:50

At some point this feature stopped working. `/esnas/autosubmit/expid/plot` folder teoretically is chmoded to 775 but it does not work.
Additionally, when creating `as_conf` object autosubmit complains of not being able to read `proj.conf` when it is `r..r..r..`.
To check...

**Comment by kinow** [comment]
In GitLab by @dmanubens on Aug 30, 2017, 11:21

Example of proj.conf problem:
```
dmanuben@ bscearth309:/esnas/autosubmit/a0p5> autosubmit monitor a0p5
Getting job list...
[CRITICAL] Unhandled exception on Autosubmit: [Errno 13] Permission denied: '/esnas/autosubmit/a0p5/conf/proj_a0p5.conf'
Traceback (most recent call last):
  File "/shared/earth/software/autosubmit/3.9.0-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.9.0-py2.7.egg/autosubmit/autosubmit.py", line 342, in parse_args
    args.filter_type, args.hide)
  File "/shared/earth/software/autosubmit/3.9.0-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.9.0-py2.7.egg/autosubmit/autosubmit.py", line 772, in monitor
    as_conf = AutosubmitConfig(expid, BasicConfig, ConfigParserFactory())
  File "/shared/earth/software/autosubmit/3.9.0-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.9.0-py2.7.egg/autosubmit/config/config_common.py", line 67, in __init__
    self.check_proj_file()
  File "/shared/earth/software/autosubmit/3.9.0-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.9.0-py2.7.egg/autosubmit/config/config_common.py", line 112, in check_proj_file
    with open(self._proj_parser_file, 'r+') as f:
IOError: [Errno 13] Permission denied: '/esnas/autosubmit/a0p5/conf/proj_a0p5.conf'
```
The snippet that fails is `open(self._proj_parser_file, 'r+') ` because `r+` is not permitted. It should be `r`. The open with + is done to be able to add a DEFAULT section on the proj conf (and supposedly to be able to treat it with config_parser). Not sure if this is really needed. An alternative could be just add the section handler in memory, not on disk.

To check also if `r+` is really needed here: 
```
platforms/ecmwf_adaptor.py:        f = open(local_file, 'r+')
```

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Nov 30, 2018, 12:44

This issue is solved in #269.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Nov 30, 2018, 12:44

closed
